src/ServicePulse.Host/app/js/directives/ui.particular.flow-diagram.js

```diff
@@ -0,0 +1,304 @@
+                    return d._children ? "lightsteelblue" : "#fff";
+                })
+                .attr('cursor', 'pointer');
+
+
+            // Remove any exiting nodes
+            var nodeExit = node.exit().transition()
+                .duration(duration)
+                .attr("transform", function(d) {
+                    return "translate(" + source.x + "," + source.y + ")";
+                })
+                .remove();
+
+            // On exit reduce the opacity of text labels
+            nodeExit.select('text')
+                .style('fill-opacity', 1e-6);
+
+            // ****************** links section ***************************
+
+            // Update the links...
+            var link = svg.selectAll('path.link')
+                .data(links, function(d) { return d.id; });
+
+            // Enter any new links at the parent's previous position.
+            var linkEnter = link.enter().insert('path', "g")
+                .attr("class", function(d){
+                    if(d.data.type === 'Event')
+                        return 'link event';
+                    else 
+                        return 'link command';
+                })
+                .attr('marker-start', 'url(#end-arrow)')
+                .attr('d', function(d){
+                    var o = {x: source.x0, y: source.y0}
+                    return straight(o, o);
+                });
+
+            // UPDATE
+            var linkUpdate = linkEnter.merge(link);
+
+            // Transition back to the parent element position
+            linkUpdate.transition()
+                .duration(duration)
+                .attr('d', function(d){ return straight(d, d.parent) });
+
+            // Remove any exiting links
+            link.exit().transition()
+                .duration(duration)
+                .attr('d', function(d) {
+                    var o = {x: source.x, y: source.y};
+                    return straight(o, o);
+                })
+                .remove();
+
+            // Store the old positions for transition.
+            nodes.forEach(function(d){
+                d.x0 = d.x;
+                d.y0 = d.y;
+            });
+            
+            function straight(s, d){
+                return 'M ' + (s.x + rectNode.width / 2) + ' ' + s.y +
+                    ' C ' + (s.x + rectNode.width / 2) + ' ' + s.y + ' ,' +
+                    (d.x + rectNode.width / 2) + ' ' + (d.y + rectNode.height - 22) + ' ,' +
+                    (d.x + rectNode.width / 2) + ' ' + (d.y + rectNode.height - 22);
+            }
+
+            // Toggle children on click.
+            function click(d) {
+                if (d.children) {
+                    d._children = d.children;
+                    d.children = null;
+                } else {
+                    d.children = d._children;
+                    d._children = null;
+                }
+                update(d);
+            }
+        }
+        if($routeParams.sergioTest)
```

Comment:
Is this part meant to make it to the final version too?

Reply:
I hope not ;) Waiting for Sergio to finish working, and removing it